modules/unsupported/geojson-core/src/main/java/org/geotools/data/geojson/GeoJSONReader.java

```diff
@@ -366,6 +366,13 @@ else if ("links".equalsIgnoreCase(name)) {
                 while (parser.nextToken() == JsonToken.START_OBJECT) {
                     ObjectNode node = mapper.readTree(parser);
                     JsonNode rel = node.get("rel");
+                    if (LOGGER.isLoggable(Level.FINE)) {
```

Comment:
This looks more like a temporary debugging aid, would it help in a deployed application?